cvr/pom.xml

```diff
@@ -65,9 +65,9 @@
     used in the pipelines importer always matches the version used by the Genome
     Nexus annotation pipeline. -->
     <dependency>
-      <groupId>com.github.genome-nexus</groupId>
+      <groupId>com.github.ao508</groupId>
```

Comment:
need to replace the `groupId` and `version` with latest master after `genome-nexus-annotation-pipeline` [PR 108](https://github.com/genome-nexus/genome-nexus-annotation-pipeline/pull/108) is merged.